src/components/dashboard/employees/Employees.js

```diff
@@ -31,16 +31,8 @@ class Employees extends Component {
   state = { loading: true };
 
   componentDidMount() {
-    this.props.getEmployees(this.props.businessId);
-  }
-
-  componentDidUpdate(prevProps) {
-    if (
-      (prevProps.employees === null && this.props.employees !== null) ||
-      (this.state.loading && this.props.employees !== null)
-    ) {
-      this.setState({ loading: false });
-    }
+    this.props.getEmployees(this.props.businessId)
```

Comment:
Are we sure we should fetch the employees any time the component updates? That could be a lot right?

Reply:
only when it mounts. github makes this look confusing but it's in the componentdidmount

Reply:
hahaha it's so late!